infra/modules/providers/azure/container-registry/tests/integration/acr.go

```diff
@@ -0,0 +1,74 @@
+//  Copyright © Microsoft Corporation
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/microsoft/cobalt/test-harness/infratests"
```

Comment:
```suggestion
	"github.com/microsoft/terratest-abstraction/integration"
```
The testing machinery (data structures and repeatable validation processes) has been lifted up into an abstraction library (https://github.com/microsoft/terratest-abstraction). All references to `infratests` would have to be replaced with `integration` after the change.